app/verify-email/page.tsx

```diff
@@ -1,67 +1,116 @@
-import React from 'react'
-import Link from 'next/link'
-import { ArrowRight } from 'lucide-react'
+'use client'
+
+import React, { useEffect, useState } from 'react'
+import { ArrowRight, CheckCircleIcon, Frown, Loader2, Mail } from 'lucide-react'
+
+function VerifyEmail() {
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const [submitSuccess, setSubmitSuccess] = useState(false)
+  const [submitFailed, setSubmitFailed] = useState(false)
+
+  const [workEmail, setWorkEmail] = useState<string | null>(null)
+
+  useEffect(() => {
+    try {
+      localStorage.getItem('workEmail')
```

Comment:
this is being not assigned to something ?

Reply:
Addressed ✅